### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by HoldenJe**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -21,10 +21,16 @@ A shiny app that provides visual representation of fish tracks within the beam a
...
-  
+2. The exported acoustic data is compiled to a master data set using [Analysis_Scripts/read_tidy_export_EVfiles.R](Analysis_Scripts/read_tidy_export_EVfiles.R). This script will import and format each fish's exported data and then compile a large dataframe *ProcessedData/processed_AllFishCombined_unfiltered.csv*.  
+3. A preliminary filtering of the data to remove values that had large TS compensation applied. This analysis [ExploratoryAnalysis/generating_filtered_compensated_targets.R](ExploratoryAnalysis/generating_filtered_compensated_targets.R)) provides a list of single target detections that have <6 dB compensation that is used to filter the dataset for analysis. The filtered file is *ProcessedData/processed_AnalysisData.csv*.  
+
+``` r
```
Comment (ross-alex) [inline README.md]: what's the purpose of having this source code here?

File: README.md [hunk 2]
```diff
@@ -21,10 +21,16 @@ A shiny app that provides visual representation of fish tracks within the beam a
...
-2. The exported acoustic data is compiled to a master data set using [Analysis_Scripts/read_tidy_export_EVfiles.R](Analysis_Scripts/read_tidy_export_EVfiles.R). This script will import and format each fish's exported data and then compile a large dataframe [ProcessedData/processed_AllFishCombined_unfiltered.csv](ProcessedData/processed_AllFishCombined_unfiltered.csv).  
-3. A preliminary filtering of the data to remove values that had large TS compensation applied. This analysis ([ExploratoryAnalysis/generating_filtered_compensated_targets.R](ExploratoryAnalysis/generating_filtered_compensated_targets.R)) provides a list of single target detections that have <6 dB compensation that is used to filter the dataset for analysis. The filtered file is [ProcessedData/processed_AnalysisData.csv](ProcessedData/processed_AnalysisData.csv).  
-  
+2. The exported acoustic data is compiled to a master data set using [Analysis_Scripts/read_tidy_export_EVfiles.R](Analysis_Scripts/read_tidy_export_EVfiles.R). This script will import and format each fish's exported data and then compile a large dataframe *ProcessedData/processed_AllFishCombined_unfiltered.csv*.
```
Comment (ross-alex) [inline README.md]: added a comment here that this process occurs locally. Could also have reflected that in the title of this subsection, but I think it's fine where I put it

File: README.md (hunk 1 above)
Comment (HoldenJe) [reply README.md]: running those two lines of code will/should? build the raw datasets now that they are no longer part of the repo as an LFS file. Each user has to clone the repo and build the large files locally.

Comment (ross-alex) [reply README.md]: I see, ya. I think it just needs some text as an explainer then - to create the above files you just need to run this code chunk in your R console

Comment (HoldenJe) [reply README.md]: how about with these additions?

Comment (ross-alex) [reply README.md]: yup, looks good to me

**Review by ross-alex — APPROVED** [review]

**Comment by HoldenJe** [comment]
closes #2

**Pull request merged**